meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/adapter/DirectChildListenerAdapter.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.helix.metaclient.impl.zk.adapter;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.List;
+import org.apache.helix.metaclient.api.DirectChildChangeListener;
+import org.apache.helix.zookeeper.zkclient.IZkChildListener;
+
+
+public class DirectChildListenerAdapter implements IZkChildListener {
+  private final DirectChildChangeListener _listener;
+
+  public DirectChildListenerAdapter(DirectChildChangeListener listener) {
+    _listener = listener;
+  }
+
+  @Override
+  public void handleChildChange(String parentPath, List<String> currentChilds) throws Exception {
```

Comment:
neat: "currentChildren"
Just curious are we planning to use this list in future?

Reply:
Thanks for the catch, will update. (Some naming and comments are carried over from I0Itech https://github.com/sgroschupf/zkclient/blob/master/src/main/java/org/I0Itec/zkclient/IZkChildListener.java, unfortunately)
We mostly likely won't need this list here, this is just to implement the interface.